manager/controllers/motion/suite_test.go

```diff
@@ -100,9 +99,9 @@ var _ = BeforeSuite(func(done Done) {
 	} else {
 		fmt.Printf("Setup fake environment... \n")
 
-		blueprintNamespace := utils.GetBlueprintNamespace()
-		fmt.Printf("Motion test suite using blueprint namespace: %s\n", blueprintNamespace)
-		workerNamespaceSelector := fields.SelectorFromSet(fields.Set{"metadata.namespace": blueprintNamespace})
+		dataAccessModuleNamespace := utils.GetDataAccessModuleNamespace()
+		fmt.Printf("Motion test suite using blueprint namespace: %s\n", dataAccessModuleNamespace)
```

Comment:
nit change comment blueprint===data access module namespace (to avoid confusion)